news/7554.removal.rst

```diff
@@ -0,0 +1,2 @@
+Removed support for VCS pseudo URLs editable requirements. It was
```

Comment:
```suggestion
Remove support for VCS pseudo URLs editable requirements. It was
```
I think we usually use the present tense for the first sentence of our changelogs.

Reply:
done